open_seq2seq/data/text2speech/text2speech_wavenet.py

```diff
@@ -0,0 +1,273 @@
+# Copyright (c) 2018 NVIDIA Corporation
+import os
+import six
+import numpy as np
+import tensorflow as tf
+import pandas as pd
+
+from open_seq2seq.data.data_layer import DataLayer
+from open_seq2seq.data.text2speech.speech_utils import \
+  get_speech_features_from_file
+
+class WavenetDataLayer(DataLayer):
+  """ Text to speech data layer class for Wavenet """
+
+  @staticmethod
+  def get_required_params():
+    return dict(
+        DataLayer.get_required_params(), **{
+            "dataset": str,
```

Comment:
Do we need "dataset" parameter? I see that it might be useful for model logging purposes, but it shouldn't be defined as a required parameter. As far as I can see, it is not used in the code currently.